src/engine/controls/enginecontrol.h

```diff
@@ -35,7 +35,6 @@ class EngineControl : public QObject {
   public:
     EngineControl(QString group,
                   UserSettingsPointer pConfig);
-    ~EngineControl() override;
```

Comment:
Overriding destructors are intentional to ensure the base class destructor is virtual.

Reply:
Could you explain that a little more? Is it basically to force subclasses to implement a destructor?